source/scenario/mission2task6.vwf.yaml

```diff
@@ -107,7 +107,6 @@ children:
             - "Use a loop to move along the slope of the tracks two more times."
           - setThirdPersonStartPose:
             - [ 16.316, 65, -15 ]
-          - startTimer:
```

Comment:
If startTimer is outside of the generic onScenarioStart (and instead in the starting VO trigger) then the timer won't change if you ever progress then move back to a scenario (given the doOnce clause at line 114).. that's what was causing the timer issues I was experiencing Friday.